src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
```

Comment:
while this css class naming looks good, we recommend to use PascalCased “blocks”, which works well with components in React.